examples/multiple-css-and-js/index.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+const http = require('http');
+
+const Tailor = require('../../');
+
+const tailor = new Tailor({
+    templatesPath: __dirname + '/templates',
+    maxAssetLinks: 2
+});
+
+// Root Server
+http.createServer((req, res) => {
+    if (req.url === '/favicon.ico') {
+        res.writeHead(200, { 'Content-Type': 'image/x-icon' } );
+        return res.end('');
+    }
+    tailor.requestHandler(req, res);
+}).listen(8080, function () {
+    console.log('Tailor server listening on port 8080');
+});
+
+// Fragment server - Any http server that can serve fragments
+http.createServer((req, res) => {
```

Comment:
Also there are lot of examples already and this doesn't need to live under a separate folder, since this is a crtiical featue, I would put this inside `multiple-fragments-with-custom-amd` folder so that we run the integration test https://github.com/zalando/tailor/blob/master/examples/multiple-fragments-with-custom-amd/test.js for the same in sauce labs.